config.py

```diff
@@ -454,23 +454,21 @@ def _setup_organisation_id(cls) -> None:
 
     @classmethod
     def _setup_members_list_auth_session_cookie(cls) -> None:
-        raw_members_list_auth_session_cookie: str | None = os.getenv(
+        raw_members_list_url_session_cookie: str | None = os.getenv(
             "MEMBERS_LIST_URL_SESSION_COOKIE",
         )
 
-        MEMBERS_LIST_AUTH_SESSION_COOKIE_IS_VALID: Final[bool] = bool(
-            raw_members_list_auth_session_cookie
-            and re.fullmatch(r"\A[A-Fa-f\d]{128,256}\Z", raw_members_list_auth_session_cookie),
+        MEMBERS_LIST_URL_SESSION_COOKIE_IS_VALID: Final[bool] = bool(
+            raw_members_list_url_session_cookie
+            and re.fullmatch(r"\A[A-Fa-f\d]{128,256}\Z", raw_members_list_url_session_cookie),
         )
         if not MEMBERS_LIST_AUTH_SESSION_COOKIE_IS_VALID:
```

Comment:
```suggestion
        if not raw_members_list_url_session_cookie or not re.fullmatch(r"\A[A-Fa-f\d]{128,256}\Z", raw_members_list_url_session_cookie):
```
Simplification for validity check which negates the undefined `IS_VALID` variable